core/src/test/java/org/apache/accumulo/core/clientImpl/ClientContextTest.java

```diff
@@ -159,10 +159,9 @@ public void testGetBatchWriterConfigNotUsingDefaults() {
         .getMemoryAsBytes(ClientProperty.BATCH_WRITER_MEMORY_MAX.getValue(props));
     assertEquals(expectedMemory, batchWriterConfig.getMaxMemory());
 
-    assertEquals(Long.MAX_VALUE, batchWriterConfig.getMaxLatency(MILLISECONDS));
+    assertEquals(40, batchWriterConfig.getMaxLatency(SECONDS));
 
-    // getTimeout returns time in milliseconds, therefore the 15 becomes 15000.
-    assertEquals(15000, batchWriterConfig.getTimeout(SECONDS));
+    assertEquals(15, batchWriterConfig.getTimeout(SECONDS));
```

Comment:
Shouldn't the units here be milliseconds?

Reply:
Same reply as above